ci-operator/step-registry/cucushift/hypershift-extended/enable-guest/cucushift-hypershift-extended-enable-guest-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
if no better idea, add more description for the kubeconfig file name used.
e.g. assume ${SHARED_DIR}/kubeconfig exists and it is mgmt cluster kubeconfig file
assume ${SHARED_DIR}/nested_kubeconfig exists and it is the hosted cluster kubeconfig file